mailpoet/lib/Abilities/WooCommerceMarketingStatus.php

```
@@ -0,0 +1,251 @@
+      if (isset($segmentsById[$segmentId])) {
+        $selectedSegments[] = $segmentsById[$segmentId];
+      }
+    }
+
+    return $selectedSegments;
+  }
+
+  private static function getAutomationEmailCounts(AutomationStorage $automationStorage): array {
+    return [
+      'abandoned_cart' => $automationStorage->getCountOfActiveByTriggerKeysAndAction([AbandonedCartTrigger::KEY], SendEmailAction::KEY),
+      'order_completed' => $automationStorage->getCountOfActiveByTriggerKeysAndAction([OrderCompletedTrigger::KEY], SendEmailAction::KEY),
```

PHPStan (2.1.54)

[error] Access to undefined constant MailPoet\Automation\Integrations\WooCommerce\Triggers\Orders\OrderCompletedTrigger::KEY.


(classConstant.notFound)